patroni/ctl.py

```diff
@@ -628,7 +628,7 @@ def failover(config_file, cluster_name, master, candidate, force, dcs, scheduled
         logging.warning('Failing over to DCS')
         click.echo(timestamp() + ' Could not failover using Patroni api, falling back to DCS')
         click.echo(timestamp() + ' Initializing failover from master {0}'.format(master))
-        dcs.manual_failover(master, candidate, scheduled_at=failover_value)
```

Comment:
Type is lost at line [596](https://github.com/zalando/patroni/blob/master/patroni/ctl.py#L596):
    scheduled_at = scheduled_at.isoformat()
It's string from now on, not date. The `isoformat` method in [dcs](https://github.com/zalando/patroni/blob/master/patroni/dcs/__init__.py#L413) will fail all the same.